LCD.c

```diff
@@ -0,0 +1,208 @@
+/*
+ * LCD.c
+ *
+ *  Created on: Feb 2, 2020
+ *      Author: helen
+ */
+
+#include "msp.h"
+#include "delay.h"
+#include "LCD.h"
+
+void init_LCD(void)                         //setup GPIO pins
```

Comment:
change the name of the function to LCD_Init. Make sure to change it in the LCD.h too